assets/libraries/terraform.rego

```diff
@@ -454,15 +454,30 @@ is_publicly_accessible(policy) {
 
 get_accessibility(resource, name, resourcePolicyName, resourceTarget) = info {
 	policy := common_lib.json_unmarshal(resource.policy)
-	info = {"accessibility": "hasPolicy", "policy": policy}
+	is_publicly_accessible(policy)
```

Comment:
We will consider that a policy is public only for looking for 'Effect' set to 'Allow' and 'Principal' set to wildcard?